modules/openapi-generator/src/main/resources/swift5/Models.mustache

```diff
@@ -56,3 +57,30 @@ extension JSONEncodable {
         self.init(statusCode: response.statusCode, header: header, body: body)
     }
 }
+
+{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} final class OpenAPIRequestCancellable {
```

Comment:
Good question, in my head this was just for request cancelation, but maybe in the future this can have more use cases.
This could possibly be used in anything that `URLSessionTask` is used.
I think you are right and we should change the name.
What about `OpenAPIRequest`?

Reply:
`OpenAPIRequest` SGTM

Reply:
I don't want to be boring, but looking at the base class RequestBuilder, and seeing that none of the other classes have the prefix OpenAPI, maybe we could name it just Request... what do you think?

Reply:
@4brunu would that cause name conflicts with the AlamoFire `Request`?

Reply:
Yes, but we can use the Alamofire.Request to disambiguate

Reply:
Giving it a second thought, I see your point and I think you are right.
Although we can use Alamofire.Request to disambiguate, it's not ideal.
What about RequestTask, given that this class is all around URLSessionTask?

Reply:
@4brunu yeh, `RequestTask` is good. you get the side-benefit that "Task" is already Swift/Foundation parlance because of the `URLSession` classes and therefore it being "cancellable" makes sense

Reply:
Done